charts/newrelic-infrastructure-beta/values.yaml

```diff
@@ -0,0 +1,398 @@
+    limits:
+      memory: 300M
+    requests:
+      cpu: 100m
+      memory: 150M
+  scraperConfig:
+    kubelet: {}
+
+
+ksm:
+  enabled: true
+  annotations: {}
+  tolerations:
+    - operator: "Exists"
+      effect: "NoSchedule"
+    - operator: "Exists"
+      effect: "NoExecute"
+  nodeSelector: {}
+  affinity:
+    podAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            topologyKey: kubernetes.io/hostname
+            labelSelector:
+              matchLabels:
+                app.kubernetes.io/name: kube-state-metrics
+          weight: 100
+    nodeAffinity: []
+  # Extra env ‘items’ that will be added to the definition of the containers; a string is expected (can be templated).
+  extraEnv:
+    - name: ENV_VAR1
+      value: "var1"
+    - name: ENV_VAR2
+      value: "var2"
+  # Extra envFrom ‘items’ that will be added to the definition of the containers; a string is expected (can be templated).
+  # extraEnvFrom:
+  #  - configMapRef:
+  #      name: 'another-config-map'
+  extraEnvFrom: []
+  # Mount additional volumes into the pod.
+  extraVolumes: []
+  # Mount additional volumes into the pod.
+  extraVolumeMounts: []
+  resources:
+    limits:
+      memory: 300M
+    requests:
+      cpu: 100m
+      memory: 150M
+  scraperConfig:
+#    interval: 10s
+#    timeout: 40s
+    ksm:
+      namespace: ""
+      port: 0
+      scheme: ""
+      selector: ""
+      staticURL: ""
+
+common:
+  scraperConfig:
+    interval: 15s
+    timeout: 30s
+
+
+# K8s DaemonSets update strategy.
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxUnavailable: 1
+
+# Kubelet's agent envs
+# Custom attributes to be passed to the New Relic agent
+customAttributes:
+  clusterName: $(CLUSTER_NAME)
+
+# Accounts and subaccounts created before July 20, 2020 will report, by default, process metrics unless this config option is explicitly set to "false"
+# On the other hand New Relic accounts created after July 20, 2020 will **not** send, by default, any process metrics unless this config option is explicitly set to "true",
+# https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1120
+# enableProcessMetrics: "false"
+
+# This can be set, the default is shown below
+# logFile: /var/log/nr-infra.log
+
+verboseLog: false
+
+serviceAccount:
+  create: true
+  annotations: {}
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# For unprivileged mode - default is 1000.
+# Can also be modified for OpenShift clusters where the runAsUser range
+# may be restricted.  See OpenShift docs for more information.
+# https://www.openshift.com/blog/a-guide-to-openshift-and-uids
+runAsUser: 1000
+# For unprivileged mode - default is 2000.
+runAsGroup: 2000
+
+### Openshift ###
+# Set openshift.enabled true when installing in an OpenShift Cluster
+openshift:
+  enabled: false
```

Comment:
Do we have any openshift-specific behavior still?

Reply:
adding the CRI-O socket

Reply:
As OpenShift is now "enough compliant" why not create a variable with the CRI so we could support differente CRI in the future instead on focusing on a distribution.